Documentation/contributing.rst

```diff
@@ -118,6 +118,7 @@ brought up by vagrant:
   cluster. Valid options are: ``docker``, ``containerd`` and ``crio``. If not
   set, it defaults to ``docker``.
 * ``VAGRANT_DEFAULT_PROVIDER={virtualbox \| libvirt \| ...}``
+* ``VM_PROXY=https://127.0.0.1:80/`` Sets up VM's ``https_proxy``.
```

Comment:
Is this supposed to be like the following?

```suggestion
* ``VM_SET_PROXY=https://127.0.0.1:80/`` Sets up VM's ``https_proxy``.
```

Reply:
Good catch, seems like it may be the case